storm-core/src/jvm/backtype/storm/Config.java

```diff
@@ -980,6 +980,126 @@
     public static final String SUPERVISOR_SLOTS_PORTS = "supervisor.slots.ports";
 
     /**
+     * What blobstore implementation the supervisor should use.
+     */
+    @isString
+    public static final String SUPERVISOR_BLOBSTORE = "supervisor.blobstore.class";
+    public static final Object SUPERVISOR_BLOBSTORE_SCHEMA = String.class;
```

Comment:
We don't use the `_SCHEMA` notation anymore with the new attribute implementation.